### DISCUSSION TITLE
해상도 분기점 관련 제의&자료 정리 포스트

### DISCUSSION THREAD

**Discussion by ryan-g00** [prgrms-fe-devcourse/NFE1_2_2_MUX#72] [D. 아이디어&레퍼런스]
공지 #71 의 포스트가 여기에요
분기점 관련 자료 같은거 좀 정리해보거나 제의하는 공간으로 씁시다

**Comment by hjlee2778** [comment]
저 분기 한 번 테스트 해봤는데요!

미디어쿼리로

데스크톱 - 태블릿 - 모바일 이렇게 해서

@media (max-width: 1023px)
@media (max-width: 767px)

이렇게 두개 설정하면 되지 않을까 싶습니다!

**Reply by ryan-g00** [reply]
/* 노트북 & 테블릿 가로 (해상도 1024px ~ )*/
@media all and (min-width:1024px) and (max-width:1279px) { 
  스타일 입력
}

/* 테블릿 가로 (해상도 768px ~ 1023px)*/
@media all and (min-width:768px) and (max-width:1023px) { 
  스타일 입력
}

/* 모바일 가로 & 테블릿 세로 (해상도 480px ~ 767px)*/
@media all and (min-width:480px) and (max-width:767px) {
  스타일 입력
}
